### PR TITLE
fix: [Go] explicitlty listen on 127.0.0.1 to avoid firewall warning p…

### PR DESCRIPTION
**Pull request by pavelgj**

### CODE REVIEW

File: go/core/servers.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func startDevServer(addr string) error {
 	// Don't use "localhost" here. That only binds the IPv4 address, and the genkit tool
```
Comment (randall77) [inline go/core/servers.go:70]: This comment needs fixing.
So "localhost" doesn't work, but "127.0.0.1" does?
(maybe take a look at the PR that added this comment)

**Review by randall77 — APPROVED** [review]

File: go/core/servers.go (hunk 1 above)
Comment (pavelgj) [reply go/core/servers.go:70]: @apascal07 do you remember?

Comment (apascal07) [reply go/core/servers.go:70]: I imagine it'll still only bind the IPv4 since it's an IPv4 address. Have you tried testing E2E?

Comment (pavelgj) [reply go/core/servers.go:70]: you mean with `genkit start`? yes, works fine.

Comment (apascal07) [reply go/core/servers.go:70]: If the dev UI works then I think the original comment can be removed.

Comment (pavelgj) [reply go/core/servers.go]: removed

**Pull request merged**